Cargo.toml

```diff
@@ -46,12 +46,15 @@ cairo-lang-semantic = { git = "https://github.com/starkware-libs/cairo", rev = "
 cairo-lang-sierra = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-sierra-generator = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-sierra-to-casm = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
+cairo-lang-sierra-type-size = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-starknet = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-starknet-classes = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-syntax = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-test-plugin = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
 cairo-lang-test-runner = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31" }
-cairo-lang-utils = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31", version = "2.5.3", features = ["env_logger"] }
+cairo-lang-utils = { git = "https://github.com/starkware-libs/cairo", rev = "7bf91178ad4a775943da53f1dab94068e5d70e31", version = "2.5.3", features = [
+    "env_logger",
+] }
```

Comment:
Where does this formatting come from?

Reply:
[Even Better TOML VSC Extension](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml)